docs/04-operation-guides/troubleshooting/00-eventing/evnt-01-eventing-troubleshooting.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Kyma Eventing - Basic Diagnostics
+---
+
+## Symptom(s)
+
+- You publish an event but the event is not received by the subscriber.
+
+- Subscription is not in ready state.
+
+- You are unable to publish events.
+
+## Remedy
+
+Follow these steps to detect the source of the problem:
+
+### Step 1: Check the status of Check the Eventing backend CR
+
+Check the Eventing backend CR, if the field `EVENTINGREADY` is true:
+   
+```bash
+kubectl -n kyma-system get eventingBackend
+```
+If `EVENTINGREADY` is false, check the exact reason of the error in the status of the CR by running the command:
+
+```bash
+kubectl -n kyma-system get eventingBackend eventing-backend -o yaml
+```
+
+### Step 2: Check the status of the Subscription
+
+Check the Subscription, if it is ready. Run the command:
+
+```bash
+kubectl -n {NAMESPACE} get subscription {NAME}
+```
+If the Subscription is not ready, check the exact reason of the error in the status of the Subscription by running the command:
+
+```bash
+kubectl -n {NAMESPACE} get subscription {NAME} -o yaml
+```
+
+If there is a problem with the Subscription sink you will see the message that sink is not valid cluster local svc. 
+Verify if the sink is a valid HTTP endpoint, for example: `test.test.svc.cluster.local`.
+
+Check if the Event type defined the Subscription is correctly formatted as specified in the [guidelines](../../../05-technical-reference/evnt-01-event-names.md).
+Also, check if the Event type is using correct `eventTypePrefix`, the event type has to start with the `eventTypePrefix`, otherwise the Subscription won't be ready.
+
+Run the following command to get the configured `eventTypePrefix` in Eventing Services:
+```bash
+kubectl get configmaps -n kyma-system eventing -o jsonpath='{.data.eventTypePrefix}'
+```
+
+### Step 3: Check if the Subscription sink is healthy.
+
+Check the workload Url defined in the Subscription sink is correct and healthy to receive events.
+To get the sink from the Subscription, run this command:
+
+```bash
+kubectl -n {NAMESPACE} get subscription {NAME} -o jsonpath='{.spec.sink}'
+```
+
+You can run the following commands to check the health of the sink:
+
+```bash
+kubectl -n default run --image=curlimages/curl --restart=Never sink-test-tmp -- curl --head {SINK_URL}
+kubectl -n default logs sink-test-tmp 
```

Comment:
I think we need to mention checking the logs of the subscriber (sink) itself in case of a none-2XX status-code is printed in the `sink-test-tmp` logs.